operations.js

```diff
@@ -506,7 +506,8 @@ Operations.prototype._sweepOps = function _sweepOps(ops, direction, callback) {
 
         var id = opKeys[i];
         var op = ops[id];
-        if (op === undefined) {
+
+        if (op === undefined && (id in ops)) {
```

Comment:
there is a race.

If the `ops[id] === undefined` then sad; if there is no `id` in `ops` then there is a race because process.nextTick and LUUUUUUUUUUUUUUUUUL.